cmd/keymasterd/app.go

```diff
@@ -253,6 +254,16 @@ var (
 	eventNotifier *eventnotifier.EventNotifier
 )
 
+func cacheControlHandler(h http.Handler) http.Handler {
+	maxAgeSeconds := maxCacheLifetime / time.Second
```

Comment:
this gives an integer? Otherwise looks good.

Reply:
Yes, both operands are of an integer type.